index.js

```diff
@@ -273,6 +273,11 @@ async function acrLogin(cred) {
 }
 
 async function dockerPush(cred) {
```

Comment:
Small typo in comment. Docket-hu

Reply:
Fixed with [0812b6d](https://github.com/choreo-templates/choreo-image-push/pull/7/commits/0812b6debc1efe3037bf8af4e5db91c9fe297cd8)